x-pack/plugins/enterprise_search/server/plugin.ts

```diff
@@ -65,7 +66,7 @@ export class EnterpriseSearchPlugin implements Plugin {
      */
     features.registerFeature({
       id: 'enterpriseSearch',
```

Comment:
Do we also want to import this id from constants?

Reply:
Oh awesome catch! Thank you!